src/curl-stream.js

```diff
@@ -0,0 +1,49 @@
+let Readable = require('stream').Readable;
```

Comment:
Easy fix: https://www.npmjs.com/package/readable-stream

Just to make sure we always have the same base-class... I'm not sure it's important, but people who understand streams seems to think this is smart :)